issue_metrics.py

```diff
@@ -307,6 +333,10 @@ def main():
 
     stats_time_to_answer = get_stats_time_to_answer(issues_with_metrics)
 
+    num_mentor_count = 0
+    if enable_mentor_count == "TRUE":
```

Comment:
```suggestion
    if enable_mentor_count:
```

my suggestion in config.py L186 will allow this to be a boolean and this can now be above code